kobo/apps/trash_bin/tasks/attachment.py

```diff
@@ -2,11 +2,15 @@
 
 from celery.exceptions import SoftTimeLimitExceeded, TimeLimitExceeded
 from celery.signals import task_failure, task_retry
+from constance import config
 from django.conf import settings
+if settings.STRIPE_ENABLED:
+    from kobo.apps.stripe.models import ExceededLimitCounter
```

Comment:
This import should not be in this block. 
BTW, don't we have a decorator for methods which require stripe to avoid this kind of conditional import?